mongo-axon-example/README.md

```diff
@@ -0,0 +1,31 @@
+# Mongo Axon Springboot Example
```

Comment:
I feel it would be valuable to place the same caveat on Mongo-as-Event-Store as we have on the [reference guide](https://docs.axoniq.io/reference-guide/extensions/mongo).
Although possible, users should not explicitly there are some downsides to using this approach for events.

For Tokens and Sagas, this is all fair game.